packages/styles/scss/components/card/_card.scss

```diff
@@ -117,6 +117,12 @@
       }
     }
 
+    .#{$prefix}--card {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+    }
```

Comment:
I have concerns about how this and #12138 might interact. Let's pause this ticket for now.